orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/RestrictExecutionDuringTimeWindowSpec.groovy

```diff
@@ -126,6 +126,27 @@ class RestrictExecutionDuringTimeWindowSpec extends AbstractBatchLifecycleSpec {
   }
 
   @Unroll
+  void 'should convert floats to integers when mapping context'() {
```

Comment:
Probably true, just wanted to verify it would actually fix the problem we're seeing. Defer to you on whether it's worth keeping in the spec